frontend/src/components/cluster-dashboard/ClusterDashboard.tsx

```diff
@@ -1,17 +1,47 @@
-import React from 'react';
+import React, { useState, useEffect, Dispatch, SetStateAction } from 'react';
+import { Grid } from '@material-ui/core';
+import axios from 'axios';
+
+import { akkaClusterProps } from './../../assets/properties/akkaClusterProps';
 import './ClusterDashboard.scss';
-import { Grid, Paper } from '@material-ui/core';
+import ClusterListView from './cluster-list-view/ClusterListView';
+import { Cluster } from './Cluster.model';
+import SimpleSnackBar from '../shared/simple-snack-bar/SimpleSnackBar';
 
 const ClusterDashboard: React.FC = () => {
+  
+  const [cluster, setCluster]: [Cluster, Dispatch<SetStateAction<Cluster>>] = useState(new Cluster());
+  const [openSnackBar, setOpenSnackBar] = React.useState(false);
+  const [snackBarMessage, setSnackBarMessage] = React.useState("");
+  
+  const loadClusterData = (): void => {
+    const akkaGetClusterMembersUrl = akkaClusterProps["akka.management.url"] + "/cluster/members";
+    const getClusterResponse = (): void => {
+      axios.get(akkaGetClusterMembersUrl)
+        .then(response => {
+          if(response.status === 200) { 
+            const clusterData: Cluster = response.data;
+            setCluster(clusterData)
+          }
+        })
+        .catch(function (error) {
+          console.log(error);
+          setSnackBarMessage(error.message && error.message.message ? error.message.message : error.message);
```

Comment:
There's a lot of `.message` going on here. As a dashboard, I don't care about what the response from the endpoint looks like, I only care what data I need to use to render. 

I would recommend these API calls be moved to their own module, where this parsing of the response and other such handling can be done separately from the consuming code, to keep the consuming code's interface to the API layer as simple as possible.